scene/gui/line_edit.cpp

```diff
@@ -1178,6 +1182,15 @@ void LineEdit::cursor_set_blink_enabled(const bool p_enabled) {
 	draw_caret = true;
 }
 
+bool LineEdit::cursor_get_force_displayed() const {
+	return caret_force_displayed;
+}
+
+void LineEdit::cursor_set_force_displayed(const bool p_enabled) {
+	caret_force_displayed = p_enabled;
```

Comment:
Need to call `cursor_set_blink_enabled(caret_blink_enabled);` to start the timer if this is toggled when out of focus.